pkgs/development/libraries/kerberos/krb5.nix

```diff
@@ -14,14 +14,23 @@ in
 with stdenv.lib;
 stdenv.mkDerivation rec {
   name = "${type}krb5-${version}";
-  majorVersion = "1.18";
+  majorVersion = "1.18"; # remove patches below with next upgrade
   version = majorVersion;
 
   src = fetchurl {
     url = "https://kerberos.org/dist/krb5/${majorVersion}/krb5-${version}.tar.gz";
     sha256 = "121c5xsy3x0i4wdkrpw62yhvji6virbh6n30ypazkp0isws3k4bk";
   };
 
+  patches = optionals stdenv.hostPlatform.isMusl [
+    # TODO: Remove with next release > 1.18
+    # Patches to fix musl build with 1.18.
+    # Not using `fetchpatch` for these for now to avoid infinite recursion
+    # errors in downstream projects (unclear if it's a nixpkgs issue so far).
```

Comment:
fetchpatch is broken in bootstrapping since fetchurlBoot doesn't support it so this is probably what you're seeing.

Reply:
@matthewbauer Is `krb5` part of bootstrapping? Should I indicate this fact here?

Reply:
Actually it's part of fetchurl definition here:

https://github.com/NixOS/nixpkgs/blob/cab6b019b17881f42f618dcc9fce9341f1a6df81/pkgs/top-level/all-packages.nix#L315-L363